### PR TITLE
Fix appName option and count billing events via dailyEvents

### PR DESCRIPTION
**Pull request by Kuchizu**
Driver 3.x only accepts lowercase appname
Limiter now sums dailyEvents daily counters for full days and counts raw events/repetitions only for the partial day around lastChargeDate

### CODE REVIEW

File: workers/limiter/src/dbHelper.ts [hunk 1]
```diff
@@ -197,6 +218,21 @@ export class DbHelper {
...
+   * @param timestamp - unix timestamp in seconds
+   */
+  private getNextUtcMidnight(timestamp: number): number {
+    const date = new Date(timestamp * MS_IN_SEC);
+
+    date.setUTCDate(date.getUTCDate() + 1);
+    date.setUTCHours(0, 0, 0, 0);
+
+    return date.getTime() / MS_IN_SEC;
+  }
```
[the remark this answers is not in the document]
Comment (neSpecc) [reply workers/limiter/src/dbHelper.ts:280]: all such corner cases should be covered by tests

File: workers/limiter/src/dbHelper.ts
Comment (neSpecc) [inline workers/limiter/src/dbHelper.ts]: I'd suggest to leave old method untouched and create the new one.

- We can opt-in new logic for a particular workspaces for testing period
- We can use old method as a fallback

They can reuse a common code

File: workers/limiter/src/index.ts [hunk 2]
```diff
@@ -328,6 +334,48 @@ export default class LimiterWorker extends Worker {
...
+   * @param since - timestamp of the time from which we count the events
+   */
+  private async getWorkspaceEventsCount(
+    workspace: WorkspaceWithTariffPlan,
+    projects: ProjectDBScheme[],
+    since: number
+  ): Promise<number> {
+    if (!this.shouldUseDailyEventsCounter(workspace._id.toString())) {
+      return this.dbHelper.getEventsCountByProjects(projects, since);
+    }
```
Comment (neSpecc) [inline workers/limiter/src/index.ts:350]: Lets compute both counters for such workspaces to be able to compare them.
The most useful way is to send logs to Telegram:

```
Workspace {name} event count:
Old algo: {old count}, took {time old}sec
New algo: {new count}, took {time new}sec
```

Comment (Kuchizu) [reply workers/limiter/src/index.ts:350]: Done

**Review by neSpecc — APPROVED** [review]
